packages/algorand-js/src/interpreter/interpreter.ts

```diff
@@ -85,6 +90,18 @@ export class Interpreter {
       for (const app of acc["created-apps"]) {
         this.globalApps.set(app.id, app.params);
       }
+
+      for (const asset of acc["created-assets"]) {
+        this.globalAssets.set(asset.index, asset.params);
+      }
+
+      const assets = acc.assets;
+      const assetInfo = new Map<number, AccountAssetInfo>();
+      for (const asset of assets) {
+        assetInfo.set(asset["asset-id"], asset);
+      }
+
+      this.accountsAssets.set(acc.address, assetInfo);
```

Comment:
as above, this logic shouldn't be in interpreter. Please add TODO to clean it up in other PR.